docs/sdks/sdk-ref-android.md

```diff
@@ -22,6 +22,8 @@ The following Android-related plugins, and associated documentation, are also av
 | To use the Google Mobile Ads (GMA) SDK to send <Link href="../ref-info/glossary-uid#gl-uid2-token">UID2 tokens</Link> as [secure signals](https://support.google.com/admob/answer/11556288) in ad requests from Android apps | [UID2 GMA Plugin for Android Integration Guide](../guides/mobile-plugin-gma-android.md) |
 | To use the Google Interactive Media Ads (IMA) SDK for Android to send <Link href="../ref-info/glossary-uid#gl-uid2-token">UID2 tokens</Link> as [secure signals](https://support.google.com/admob/answer/11556288) in ad requests from Android apps | [UID2 IMA Plugin for Android Integration Guide](../guides/mobile-plugin-ima-android.md) |
 
+For FAQs relating to mobile publisher integrations, see [FAQs for Mobile Integrations](../guides/integration-mobile-overview.md#faqs-for-mobile-integrations).
```

Comment:
Should we have a standalone FAQ section to contain this paragraph and make it obvious and link it to the FAQs for Mobile Page?

This should also be mentioned in
[UID2 Client-Side Integration Guide for Mobile](https://unifiedid.com/docs/guides/integration-mobile-client-side)
[UID2 Client-Server Integration Guide for Mobile](https://unifiedid.com/docs/guides/integration-mobile-client-server)

Reply:
@sunnywu on your first question: I just don't think we're at critical mass yet. Plus generally the FAQs are grouped by audience so I guess it should go in the publisher section. At this point I think we should keep it where it is. Don't want to put it in two places.

Added a link to this doc in the other two guides stated.